src/store/store.ts

```diff
@@ -412,6 +428,22 @@ export class Store {
 		(this.projects as IObservableArray<Project>).remove(project);
 	}
 
+	/**
+	 * Renames the name of the pages files and update the names
+	 * @param id The new ID of the page.
+	 */
+
+	public renameYaml(id: string): void {
```

Comment:
`renamePage` would be a more descriptive name :)

Reply:
@LKuechler  I will do change this. Got it :)

Reply:
@LKuechler  Done ✅